ibis/expr/schema.py

```diff
@@ -9,6 +9,28 @@
 import ibis.expr.datatypes as dt
 import ibis.util as util
 
+convert = Dispatcher(
+    'convert',
+    doc="""\
+Convert `column` to the pandas dtype corresponding to `out_dtype`, where the
+dtype of `column` is `in_dtype`.
+
+Parameters
+----------
+in_dtype : Union[np.dtype, pandas_dtype]
+    The dtype of `column`, used for dispatching
+out_dtype : ibis.expr.datatypes.DataType
+    The requested ibis type of the output
+column : pd.Series
+    The column to convert
+
+Returns
+-------
+result : pd.Series
```

Comment:
A bit confused.. Does return a dtype or a pd.Series?

Reply:
I just moved this function of file without change, probably better to fix things in a follow up. I can have a look since this is just the docs, but in general better to not make changes that don't show in a commit diff.

Reply:
kk SGTM